templates/quickstart-jira-dc.template.yaml

```diff
@@ -523,7 +523,7 @@ Parameters:
     Type: String
   JiraProduct:
     Default: Software
-    Description: The Jira product to install. If "All" is selected, Jira Software and Jira Service Management will be installed
+    Description: The Jira product to install. If "All" is selected, Jira Software and Jira Service Management will be installed but you will need to paste in a Jira Service Management license after configuring Jira Software.
```

Comment:
Or we can say: `. A Jira Service Management license is required after configuring Jira Software.`

Reply:
So the full string would be:
> The Jira product to install. If "All" is selected, a Jira Service Management license is required after configuring Jira Software.
?

Reply:
I meant :
`The Jira product to install. If "All" is selected, Jira Software and Jira Service Management will be installed. A Jira Service Management license is required after configuring Jira Software.`

but your rephrase even looks better :)

Reply:
I prefer your wording, it's a bit more verbose but it's clear.